mlflow/store/dbmodels/initial_artifact_store_models.py

```diff
@@ -0,0 +1,66 @@
+# Snapshot of MLflow DB models as of the 0.9.1 release, prior to the first database migration.
+# Used to standardize initial database state.
+# Copied with modifications from
+# https://github.com/mlflow/mlflow/blob/v0.9.1/mlflow/store/dbmodels/models.py, which
+# is the first database schema that users could be running. In particular, modifications have
+# been made to substitute constants from MLflow with hard-coded values (e.g. replacing
+# SourceType.to_string(SourceType.NOTEBOOK) with the constant "NOTEBOOK") and ensure
+# that all constraint names are unique. Note that pre-1.0 database schemas did not have unique
+# constraint names - we provided a one-time migration script for pre-1.0 users so that their
+# database schema matched the schema in this file.
+
+from sqlalchemy import (
+    Column, String, VARBINARY, BigInteger, Integer, PrimaryKeyConstraint)
+from sqlalchemy.ext.declarative import declarative_base
+
+from mlflow.entities import FileInfo
+import os
+
+Base = declarative_base()
+
+
+class SqlArtifact(Base):
```

Comment:
Will this be added to all MLflow SQL store databases? Or do users have to opt in?

Reply:
@mateiz Do you mean the backend SQL database where runs are recorded? This table will not be added there. Only when the users specify a RDBMS as an artifact store, this table will be created in the RDBMS.